packages/account/src/Components/terms-of-use/terms-of-use.tsx

```diff
@@ -11,10 +11,45 @@ import {
 } from '@deriv/components';
 import { isDesktop, isMobile, PlatformContext } from '@deriv/shared';
 import { localize, Localize } from '@deriv/translations';
-import CheckboxField from './checkbox-field.jsx';
-import { SharedMessage, BrokerSpecificMessage, Hr } from './terms-of-use-messages.jsx';
+import { TBrokerCodes } from 'Types';
+import CheckboxField from './checkbox-field';
+import { SharedMessage, BrokerSpecificMessage, Hr } from './terms-of-use-messages';
 import './terms-of-use.scss';
 
+type TTermsOfUseFormProps = {
+    agreed_tos: boolean;
+    agreed_tnc: boolean;
+};
+
+type TTermsOfUseProps = {
+    getCurrentStep: () => number;
```

Comment:
the list is not huge but anyway, wouldn’t it be better to sort all fields alphabetically? :)